qdrant-landing/content/documentation/tutorials/neural-search.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Create a Simple Neural Search Service
+weight: 14
+---
+
+# Create a Simple Neural Search Service
+
+| Time: 30 min | Level: Beginner | Output: [GitHub](https://github.com/qdrant/qdrant_demo/blob/master/qdrant_demo/init_vector_search_index.py) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1kPktoudAP8Tu8n8l-iVMOQhVmHkWV_L9?usp=sharing)   |
+| --- | ----------- | ----------- |----------- |
+
+This tutorial shows you how to build and deploy your own neural search service. The created service will be searching through descriptions of companies from [startups-list.com](https://www.startups-list.com/). The website contains the company names, descriptions, their locations and a picture for each entry. 
```

Comment:
Do you think it might be useful to have a minimal illustration of what we are building here?
Maybe a small diagram from the https://excalidraw.com/ ?

I understand that you don't want theory here, but can this diagram help a bit? In future it might make sense to ask actual designer to draw those

Reply:
no problem, will add something minimal, but it will show the relationships